src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/AxSystemMonitorTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.ComponentModel;
+
+namespace System.Windows.Forms.Tests;
+
+public class AxSystemMonitorTests
+{
+    private readonly Form form;
+    private readonly AxSystemMonitor.AxSystemMonitor control;
```

Comment:
Fields should have `_` prefix
```suggestion
    private readonly Form _form;
    private readonly AxSystemMonitor.AxSystemMonitor _control;
```